components/sensor/nextion.rst

```diff
@@ -0,0 +1,176 @@
+.. _nextion_sensor:
+
+Nextion Sensor Component
+========================
+
+.. seo::
+    :description: Instructions for setting up Nextion sensor.
+    :image: nextion.jpg    
+
+The ``nextion`` sensor platform supports intergers or floats (Xfloat). It can be a component, a variable or a waveform in the Nextion display.
```

Comment:
```suggestion
The ``nextion`` sensor platform supports integers or floats (Xfloat). It can be a component, a variable or a waveform in the Nextion display.
```